who-is-openstreetmap/index.md

```diff
@@ -5,4 +5,5 @@ order: 1
 color: blue
 description: The people and communities of OSM
 image: whatishistory_14665124541_52ce01db2c_o_KLL.jpg
+image-alt: a group of people sitting on stairs
```

Comment:
@russbiggs maybe "a group of people posing on stairs after completing a mapping workshop"